ColorPairTest.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+#include <iostream>
+#include <assert.h>
+#include "ColorPair.h"
+#include "ColorFormatter.h"
+
+void testNumberToPair(int pairNumber,
+    TelCoColorCoder::MajorColor expectedMajor,
+    TelCoColorCoder::MinorColor expectedMinor)
+{
+    TelCoColorCoder::ColorPair colorPair =
+        TelCoColorCoder::ColorPair::GetColorFromPairNumber(pairNumber);
+    std::cout << "Got pair " << colorPair.ToString() << std::endl;
+    assert(colorPair.getMajor() == expectedMajor);
+    assert(colorPair.getMinor() == expectedMinor);
+}
+
+void testPairToNumber(
+    TelCoColorCoder::MajorColor major,
+    TelCoColorCoder::MinorColor minor,
+    int expectedPairNumber)
+{
+    int pairNumber = TelCoColorCoder::ColorPair::GetPairNumberFromColor(major, minor);
+    std::cout << "Got pair number " << pairNumber << std::endl;
+    assert(pairNumber == expectedPairNumber);
+}
+
+void testColorCodeFormatterWithValidTotalcolor()
+{
+    std::string colorpairStr = ColorFormatter::printcolorcodes();
+    std::cout << "\n\n";
+    std::cout << "Color Code formatter value " << colorpairStr << std::endl;
+    assert(!colorpairStr.empty());

```

Comment:
Good, when this test passes, it says "the code doesn't crash and gives non-empty report"
This is what we should aim for - simple tests that remove some load off manual testing (rather than having no tests at all, because alignment and all are difficult to test)